.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+<!--
+List the issues this PR closes (if any) in a bullet list format, e.g.:
+- Closes #ABCD
+- Closes #EFGH
+-->
+
```

Comment:
Could we also have an optional section for high-level descriptions?

I did it manually in [this case](https://github.com/FuelLabs/fuels-ts/releases/tag/v0.91.0), but it could be one point coming from each PR.

<details>
  <summary>Screenshot</summary>

![summary](https://github.com/FuelLabs/fuels-ts/assets/26660/541bf206-2c9f-456d-a930-bdc16479e85d)

</details>

This would require a new section or something similar.

```suggestion
# Release Notes

<!--
Use this only if this PR requires a mention in the Release Notes Summary.
Valuable features and critical fixes are good examples.
For everything else, please delete the whole section.
-->

In this release, we:
 - Did this and that <!-- edit this text only -->

```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2688/commits/8b4c21a3034572b59a6d33b93fd3d11097067ad3

I just requested a bullet point list without the `In this release, we:\n` because it's easier to parse.

Reply:
The `In this release, we:` provides context and ensures the verb tense for the bullet point.

Without it, determining how to frame the phrase may be tricky and yield incoherent updates in the release notes.

Reply:
Great foresight! https://github.com/FuelLabs/fuels-ts/pull/2688/commits/fa22c53648a6e10cda2ef29bd7b2070ebb0e858b